<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlEditDialog</class>
 <widget class="QWidget" name="SqlEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>栈板限高:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_XPos">
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>中心坐标Z:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_SkuCode"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>栈板宽度:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>中心坐标X:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_YPos">
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="spinBox_FloorNum"/>
        </item>
        <item row="8" column="2">
         <widget class="QSpinBox" name="spinBox_Angel">
          <property name="maximum">
           <number>359</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>层数:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>SKU码:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>箱子进入方向：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>栈板长度:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>中心坐标Y:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QSpinBox" name="spinBox_Direction">
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>箱子朝向:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_ZPos">
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_TrayLength"/>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_TrayWidth"/>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="spinBox_TrayHeight"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
